src/components/choose-locale/change-locale.component.tsx

```diff
@@ -4,6 +4,7 @@ import SelectItem from "carbon-components-react/es/components/SelectItem";
 import { refetchCurrentUser } from "@openmrs/esm-framework";
 import { updateUserProperties } from "./change-locale.resource";
 import styles from "./changelocal.component.scss";
+import { Dropdown } from "carbon-components-react";
```

Comment:
Please use the import from its submodule to avoid bundling in everything from CCR.